### ISSUE TITLE
Option to jump to definitions?

### ISSUE THREAD

**Issue by gamb** [psibi/dhall-mode#29]
Short of digging into `dhall-lsp-server`, the cheapest option I could find to hop between _files_ was:
```elisp
(defun jump-to-file-at-point ()
  (interactive)
  (ring-insert find-tag-marker-ring (point-marker))
  (find-file-at-point))
```

**Comment by psibi** [comment]
I guess a much better way would be to add support for dhall in the dumb-jump package: https://github.com/jacktasia/dumb-jump

**Comment by gamb** [comment]
@psibi Yeah that sounds good to me :+1: I'll give it a go when I get some time

**Comment by gamb** [comment]
@psibi Ahh, TIL... `dhall tags` works fairly well

**Comment by psibi** [comment]
@gamb I guess adding support based to dhall-mode based on `dhall tags` would be a nice approach.
